internal/pkg/config/defaults/gt10000_limits.yml

```diff
@@ -1,26 +0,0 @@
-num_agents:
-  min: 10000
-  max: 12499
```

Comment:
Turns out this setting was never changed and actually leads to a max agents val of 12500 using default values, incorrectly causing one of the previous unit tests to pass.
My current changes have caused the test to fail.